tts-core/src/main/java/org/mobicents/media/server/tts/core/LoadTTSEngine.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * TeleStax, Open Source Cloud Communications
+ * Copyright 2011-2016, Telestax Inc and individual contributors
+ * by the @authors tag. 
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.mobicents.media.server.tts.core;
+
+import java.io.File;
+import java.lang.reflect.InvocationTargetException;
+import java.net.URL;
+
+import org.apache.commons.configuration.XMLConfiguration;
+import org.apache.log4j.Logger;
+import org.jboss.util.StringPropertyReplacer;
+
+
+/**
+ * LoadTTSEngine which parse configuration file to load TTS engine and handle text from user
+ *   
+ * @author Thanh Tran
+ *
+ */
+public class LoadTTSEngine {
```

Comment:
The purpose of this class seems wrong. It should only load configuration and build the TTS Engine. It should not actually do TTS operations. Please decouple.